packages/openstellina/openstellina.opam-publish-openstellina.0.0.3/opam

```diff
@@ -0,0 +1,143 @@
+opam-version: "2.0"
+synopsis: "A http client for Stellina smart telescope by Vaonis"
+description:
+  "Mobile Stellina App does not provide access to telescope exposures > 20s (lights or darks). This software bypasses the App and talks to the telescope directly"
+maintainer: ["Jonathan Kimmitt"]
+authors: ["Jonathan Kimmitt"]
+license: "Apache-2.0"
+available: arch != "arm32" & arch != "x86_32"
+tags: ["Vaonis" "Stellina" "long-exposure" "GUI"]
+homepage: "https://github.com/jrrk2/openstellina.git"
+bug-reports: "https://github.com/jrrk2/openstellina.git/issues"
+depends: [
```

Comment:
This is a really long list, are you sure you need them all?
The dune file only links against `lablgtk2 lwt_glib yojson cohttp-lwt-unix zeit ppx_let quests ppx_yojson lwt`

Reply:
with v0.0.2 I only included direct dependents. But this fails downgrade checks because some indirect dependents are fragile. I generated this list from a fresh switch.

Reply:
Very happy to rename my folders according to your preference.

Reply:
To be fair, I'd rather see the package only with the needed dependent libraries, and then fix the broken lower bounds with the appropriate conflicts if needed.

If it goes wrong when you only specify dune, ocaml, lablgtk2 lwt_glib, yojson, cohttp-lwt-unix, zeit, ppx_let, quests, ppx_yojson and lwt, something should be off in their opam files and we should fix it

Reply:
Perhaps not entirely true because some indirect dependencies such as let_ssl affect the compilation of direct dependencies. Anyway, I think my list is fairly short now and I would be happy to go with it.